airbyte-cdk/java/airbyte-cdk/core/build.gradle

```diff
@@ -102,4 +105,8 @@ dependencies {
     testImplementation 'org.apache.commons:commons-lang3:3.11'
     testImplementation 'org.xerial.snappy:snappy-java:1.1.8.4'
     testImplementation 'org.mockito:mockito-core:4.6.1'
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
 }
+
+compileKotlin {}
```

Comment:
You probably want to set the `jvmTarget` here (and probably `languageVersion` as well):
```groovy
compileKotlin {
  compilerOptions {
    jvmTarget = "21" // or whatever version of java this code is using
    languageVersion = "1.9"
  }
}
```

Reply:
This could also be replaced with a `toolchain` in gradle, but I don't believe the connectors are currently using `toolchains` anywhere.  

The platform has migrated to the `toolchain` approach.